packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -67,20 +47,14 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
   onFetchData,
   onRowClick,
   totalLength = 0,
-}: TableProps<T> & { children?: ReactNode }): JSX.Element => {
+  tableApi,
+}: TableProps<T>): JSX.Element => {
```

Comment:
played around with `forwardRef` but couldn't keep the nice generic typing and also made a little more sense to name the prop something other than `ref` (although unsure if `tableApi` is that much better.. )